addon/mixins/scoped-query-cache.js

```diff
@@ -30,29 +42,19 @@ export default Ember.Mixin.create({
           formatCacheKey(query) :
           query;
 
-    assert('queryCache needs to be an object', Ember.typeOf(queryCache) === 'object');
+    assert('queryCache needs to be an object', typeOf(queryCache) === 'object');
     return JSON.stringify(sortedObject(queryCache));
   },
 
-  queryCache(modelName, query, options = {}, queryMethod) {
+  getFromCache(modelName, query, options = {}, queryMethod) {
```

Comment:
So I think there's two layers here. The first is the namespaced functionality added to the mixin. Whether that's `queryCache` or `scopedStore` is TBD. I do want to keep the query functionality separate though, as the query methods that pertain to the store seem a separate concern that having a map and storing values in it. I'll likely break these out into separate classes.

Thoughts?